ocaml/xapi/nm.ml

```diff
@@ -300,6 +300,7 @@ let rec create_bridges ~__context pif_rc net_rc =
     let ports = [pif_rc.API.pIF_device, {default_port with interfaces=[pif_rc.API.pIF_device]}] in
     cleanup,
     [net_rc.API.network_bridge, {default_bridge with ports; bridge_mac=(Some pif_rc.API.pIF_MAC);
+                                                     igmp_snooping=(Some (Db.Pool.get_igmp_snooping_enabled ~__context ~self:(Helpers.get_pool ~__context)));
```

Comment:
Please do the DB-get on a separate line so that it is easier to read.
This needs to be done in `create_bond` as well.